hack/release.sh

```diff
@@ -0,0 +1,96 @@
+#!/bin/sh
+#
+# Cut a libpod release.  Usage:
+#
+#   $ hack/release.sh
+
+DATE=$(date +%Y-%m-%d)
+MONTH=$(echo "${DATE}" | sed 's/[0-9]*-0\?\([0-9]*\)-[0-9]*/\1/')
+NEXT_MONTH=$(date --date 'next friday' +%m | sed 's/0\?\([0-9]*\)/\1/')
+LAST_TAG=$(git describe --tags --abbrev=0)
+LAST_MONTH=$(echo "${LAST_TAG}" | sed -n 's/v0\.\([0-9]*\)\.[0-9]*/\1/p')
+if test -z "${LAST_MONTH}"
+then
+	echo "unable to parse month from ${LAST_TAG}" >&1
```

Comment:
I think you want 1>&2 here, as written your writing stdout to stdout.

Reply:
> I think you want 1>&2 here...

Good catch, but that line is gone now that we're [back to getting these from the command line][1].

[1]: https://github.com/projectatomic/libpod/pull/867#issuecomment-393756522